sdk/core/azure-core-http-httpurlconnection/src/main/java/com/azure/core/http/httpurlconnection/HttpUrlConnectionClientProvider.java

```diff
@@ -0,0 +1,11 @@
+package com.azure.core.http.httpurlconnection;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpClientProvider;
+
+public class HttpUrlConnectionClientProvider implements HttpClientProvider {
```

Comment:
Refer to the other provider implementations - we want to have configuration read in here, e.g. `AZURE_ENABLE_HTTP_CLIENT_SHARING`